selftests/functional/resolver.py

```diff
@@ -140,5 +142,42 @@ def test_runnable_recipe(self):
         )
 
 
+class ResolverFunctionalTmp(TestCaseTmpDir):
+    def test_runnables_recipe(self):
+        all_runnable_recipes = glob.glob(
```

Comment:
What about create a runnables_recipe_file in examples instead of generating one from the other recipes? IMO, this test would be easier and users will have a working example. You can also use this example inside documentation.

Reply:
You have a point, but then, the counterargument is: it can get out of sync pretty fast.  Meaning, that if we add fields or newer examples to the "runnable", the "runnables" would fall behind.  I think that for testing purposes, having it generated (and in sync) results in better testing.